nodes/nodes/rosa-working-with-nodes.adoc

```diff
@@ -0,0 +1,17 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="rosa-working-with-nodes"]
+= Working with nodes
+include::_attributes/attributes-openshift-dedicated.adoc[]
+
+:context: rosa-working-with-nodes
+
+toc::[]
+
+As a {product-title} (ROSA) administrator, you can perform several tasks to make your clusters more efficient. You can use the `oc adm` command to cordon, uncordon, and drain a specific node. This is available for both ROSA Classic and {hcp-title-first} clusters.
+
+[NOTE]
+====
+Cordoning and draining are only allowed on worker nodes that are part of OCM machine pools.
+====
```

Comment:
Is there any way that we can add another note here around cordoning and draining nodes potentially affecting User Workload Monitoring?

Essentially the problem is that if, say, a customer only has two worker nodes in the cluster and they cordon and drain one their User Workload Monitoring will be degraded and any node instability on the other node could cause issues with UWM.

We have a PR open that will automatically send a Service Log in this case when this is happening, but I think it's also worth mentioning in the docs.

Reply:
Additionally - it looks like we may also want to call out a potential impact to upgrades, as well as impact to workloads through an upgrade.

Using the previous example, if a customer has cordoned a node and attempts to schedule an upgrade but there's a critical alert firing because of the node being unshcheduleable the upgrade attempt will fail. Additionally, if there is NOT an alert firing and the upgrade progresses it could cause potential disruptions to workloads if the node is already cordoned, or the node may eventually be uncordoned by the upgrade process - I'm not sure exactly what would happen.

Reply:
> Is there any way that we can add another note here around cordoning and draining nodes potentially affecting User Workload Monitoring?

Kirk, Since UWM is customer managed workload, I'd say it is not necessary to call this out. This is applicable and hence true for any workload they run. OC ADM commands are meant for administrators (not developers). So, they should be aware of what they're doing. Cordon/Drain is a standard K8S tool so I think we will be fine. 

>  it looks like we may also want to call out a potential impact to upgrades

We don't need this as well because 1) we are working on doing Upgrade preflight checks in MUO and validating this. Additionally, after 60 minutes of node being cordoned, we are notifying. So, we don't need to make documentation complicated with lot of warnings.

Reply:
Appreciate reviewing and bring up these concerns.

Reply:
That's fair - We've been supporting UWM as a managed workload though from the SRE standpoint - we get paged when it's down and everything (even due to customer misconfiguration, which is most often the case). Are you saying that we can push the onus of monitoring UWM to the customer and we can stop paging SRE on UWM alerts entirely?

Reply:
This is news to me. Yes, we must not be paging SRE for that. Let's discuss offline.